src/theme.ts

```diff
@@ -12,19 +12,22 @@ const palette = {
     // green colors
     main: "#006778",
     light: "#c2dbdf",
+    lighter: "#c2dbdf",
     lightest: "#e5eff1",
   },
   secondary: {
     // grey colors
     main: "#595959",
     light: "#dfe1e3",
+    lightest: "#DFE1E3",
   },
   background: {
-    default: "#e5eff1",
+    default: "#d8e8ea",
     paper: "#fff",
   },
   success: {
-    main: "#51a808",
```

Comment:
I think we still need to keep the color `#51a808` as it is used in snack bar in the design.

![Screenshot 2022-04-22 at 15 24 16](https://user-images.githubusercontent.com/23532204/164714446-1f018c4e-1d3b-421a-92b2-cfae8670c3b0.png)

Reply:
Yes, `success` color values are now extended so this color can be kept.